src/panels/lovelace/cards/hui-media-control-card.ts

```diff
@@ -84,106 +105,168 @@ export class HuiMediaControlCard extends LitElement implements LovelaceCard {
 
     const picture = this._image || "../static/images/card_media_player_bg.png";
 
+    const imageStyle = {
```

Comment:
Why declare this as a variable if it's only used once ?

Reply:
Readability. as well as VScode doesn't seem to like multiple lines of style with JS in the template